dubbo-metrics/dubbo-metrics-default/src/main/java/org/apache/dubbo/metrics/filter/MethodMetricsInterceptor.java

```diff
@@ -85,6 +86,7 @@ public void handleMethodException(Invocation invocation, Throwable throwable) {
                 eventType = MetricsEvent.Type.NETWORK_EXCEPTION;
             }
         }
+
```

Comment:
if (e.isBiz()) {
                eventType = MetricsEvent.Type.BUSINESS_FAILED;
            }
          Can this code be deleted directly？

Reply:
This might throw by Filter by custom